src/StateEither.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * @since 2.4.4
+ */
+import * as E from 'fp-ts/lib/Either'
+import { Monad3 } from 'fp-ts/lib/Monad'
+import { MonadThrow3 } from 'fp-ts/lib/MonadThrow'
+import { State } from 'fp-ts/lib/State'
+import { getStateM } from 'fp-ts/lib/StateT'
+import { pipeable } from 'fp-ts/lib/pipeable'
+
+const T = getStateM(E.either)
+
+declare module 'fp-ts/lib/HKT' {
+  interface URItoKind3<R, E, A> {
+    StateEither: StateEither<R, E, A>
+  }
+}
+
+/**
+ * @since 0.1.0
+ */
+export const URI = 'StateEither'
+
+/**
+ * @since 0.1.0
+ */
+export type URI = typeof URI
+
+/**
+ * @since 0.1.0
+ */
+export interface StateEither<S, E, A> {
+  (s: S): E.Either<E, [A, S]>
+}
+
+/**
+ * @since 0.1.0
+ */
+export function run<S, E, A>(ma: StateEither<S, E, A>, s: S): E.Either<E, [A, S]> {
```

Comment:
This function is useless, let's remove it (there's no `run` function in the `State` module)